AppAuth-JS/built/authorization_request_handler.d.ts

```diff
@@ -0,0 +1,63 @@
+import { AuthorizationRequest } from './authorization_request';
```

Comment:
This is generated source file. All the files in built/ folder are generated JS file for the Google's lib fork